aspnetcore/razor-pages/index.md

```diff
@@ -191,7 +191,7 @@ Reviewing the *Pages/Create.cshtml* view file:
 [!code-cshtml[](index/3.0sample/RazorPagesContacts/Pages/Customers/Create.cshtml?highlight=3,9)]
 
 * In the preceding code, the [input tag helper](xref:mvc/views/working-with-forms#the-input-tag-helper) `<input asp-for="Customer.Name" />` binds the HTML `<input>` element to the `Customer.Name` model expression.
-* [`@addTagHelper`](xref:mvc/views/tag-helpers/intro#addtaghelper-makes-tag-helpers-available) makes Tag Helpers available.
+* [@addTagHelper](xref:mvc/views/tag-helpers/intro#addtaghelper-makes-tag-helpers-available) makes Tag Helpers available.
```

Comment:
prefer original. Hopefully this renders OK

Reply:
Same deal here ... ~45 occurrences without styling. This is perhaps the only one **with** styling.